backend/src/appstate.rs

```diff
@@ -1,21 +1,52 @@
 use actix::Addr;
 
+use serde::Serialize;
 use std::sync::Mutex;
 
+use crate::api;
+use crate::communicator;
 use crate::game;
 use crate::websocket::Websocket;
 
 #[derive(Debug)]
 pub struct AppState {
     pub clients: Mutex<Vec<Addr<Websocket>>>,
     pub game: Mutex<game::Game>,
+    pub current_player: Mutex<u32>,
```

Comment:
I think the communicator should encapsulate all these fields into itself, since it is, in fact, a game server.
This way we could write game-related logic in the appropriate place.
Obviously, `next_turn` is game logic, not an application as a whole.

This approach will also let us leave only one mutex, for the server itself, which would be nice.

Reply:
We have to name this abstraction something like "room" or "lobby" 

And thus everything become correct)) And when we'll have several games, we can return appstate and it will contain vector of lobbies

Reply:
I think that lobby is one thing, and current games in progress is another.

Reply:
https://github.com/Miol-Mor/battle-game/issues/74